chart/fluentbit-operator/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.serviceAccount.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{  .Values.metadata.name  }}
+  namespace: {{  .Values.metadata.namespace  }}
```

Comment:
The `namespace` should not be set, it will be set by helm if the helm running with `-n`.